src/Sulu/Component/Rest/FlattenExceptionNormalizer.php

```diff
@@ -64,6 +66,16 @@ public function normalize($exception, $format = null, array $context = [])
             $data['errors'] = [$errors];
         }
 
+        if ($contextException instanceof DeletionImpossibleChildPermissionsException) {
```

Comment:
we should create an Interface for it like we did for the `TranslationErrorMessageExceptionInterface`